src/mito2/src/request.rs

```diff
@@ -596,7 +595,7 @@ pub(crate) struct FlushFinished {
     /// File purger for cleaning files on failure.
     pub(crate) file_purger: FilePurgerRef,
     /// Flush timer.
-    pub(crate) timer: Timer,
+    pub(crate) timer: Option<HistogramTimer>,
```

Comment:
Because the `timer` in the `request` will be dropped first, obtain the elapse time and output to the log, but then an ownership error will occur when calling the `on_succes` method of the `request`. Because the `timer` has been dropped, so a trick is taken `Some(timer)`  in the request and replaced  with `None` , thus ensuring the ownership of the `request` when `on_succes` is called

Reply:
I thought `timer` needed to end there, it was written like this to avoid rust ownership, it can be removed now